test/es-module/test-esm-loader-hooks.mjs

```diff
@@ -517,6 +517,37 @@ describe('Loader hooks', { concurrency: true }, () => {
       assert.strictEqual(signal, null);
     });
 
+    it('should have `register` accept URL objects as `parentURL`', async () => {
```

Comment:
It seems like the code you added also allows URL instances as the first argument, too; should we also add a test for that?

I assume the docs probably also need to be updated too that these arguments can be either strings or URLs.

Reply:
It was already the case, and it's already covered by tests, e.g. here: https://github.com/nodejs/node/blob/a40a6c890afe0d1d0ca78db015146178c25af079/test/fixtures/es-module-loaders/register-loader.cjs#L4

Reply:
Okay but looking at `module.md` I think we still need to update the docs to say that URLs are allowed.

There’s also the `parentURL` property of the options object.